specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2018-03-01-preview/machineLearningServices.json

```diff
@@ -500,7 +500,6 @@
           "MachineLearningComputes"
         ],
         "operationId": "MachineLearningCompute_CreateOrUpdate",
-        "x-ms-long-running-operation": true,
```

Comment:
Is this header not used anymore?

Reply:
added long-running back. Update accordingly